src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav">
+        <div class="nav__logo">
+          <a
+            href="#"
+            class="logo__link"
+          >
+            <img
+              src="./images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active under-line"
```

Comment:
The class `is-active` suggests that this link is the currently active page. Ensure that this class is dynamically applied to the correct link based on the current page context.